packages/flutter/lib/src/services/mouse_cursor.dart

```diff
@@ -191,9 +193,9 @@ abstract class MouseCursorSession {
 /// a cursor, and defines the states and behaviors of the cursor. Every mouse
 /// cursor class usually has a corresponding [MouseCursorSession] class.
 ///
-/// [MouseTrackerCursorMixin] is a mixin that adds the feature of changing
-/// cursors to [BaseMouseTracker], which tracks the relationship between mouse
-/// devices and annotations. [MouseTrackerCursorMixin] is usually used as a part
+/// [MouseCursorManager] is a mixin that adds the feature of changing
```

Comment:
```suggestion
/// [MouseCursorManager] is a class that adds the feature of changing
```

If it's really a mixin, it should be declared as `mixin`.